README.md

```diff
@@ -1,87 +1,90 @@
-<a href="https://www.buildwithfern.com/">
-  <img src="header.png" alt="header" />
-</a>
-
+<br/>
 <div align="center">
-  <a href="https://www.buildwithfern.com/docs/intro" alt="documentation">Documentation</a>
-  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://discord.com/invite/JkkXumPzcG" alt="discord">Join the Discord</a>
-</div>
+  <a href="https://www.buildwithfern.com/">
+    <img src="fern.png" height="120" align="center" alt="header" />
+  </a>
+  <br/>
 
-<br />
+# Fern
+
+![YC](https://img.shields.io/badge/Y%20Combinator-2023-orange)
+![License](https://img.shields.io/badge/License-MIT-blue)
+
+[![Discord](https://img.shields.io/badge/Join%20Our%20Community-black?logo=discord)](https://discord.com/invite/JkkXumPzcG)
+[![Documentation](https://img.shields.io/badge/Read%20our%20Documentation-black?logo=book)](https://www.buildwithfern.com/docs/intro)
 
-<div align="center">
-Backed by Y Combinator
 </div>
 
----
+Fern is an open source toolkit for designing, building and consuming REST APIs. With Fern, you can generate client libraries, documentation and boilerplate for your backend server in your chosen framework (e.g. FastAPI, Express).
+
+Fern can be used in your existing projects through OpenAPI or if you choose, you can describe your API in the Fern specification. In this sense, Fern is an alternative to the OpenAPI standard but does not aim to replace it.
+
+Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering value**.
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Capabilities
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+Currently, the Fern compiler can generate the following types of artifacts:
 
-### 🌿 SDKs
+### 🌿 Client Libraries and SDKs
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+Whether you call them SDKs, client libraries, bindings, or wrappers, Fern generates idiomatic SDKs you can use to interact with your APIs, or publish for third party usage. This makes keeping your client libraries up to date with your API endpoints trivial, and speeds up your workflows dramatically.
```

Comment:
I think in this para we want to: 
- clarify that what we call SDK could also be referred to as client library, binding, wrapper
- mention that we've focused on the quality of the generated code (as if a human wrote it)
- generated SDKs contain types, networking logic, error handling so that you can *consume* the API effortlessly